sklearn/_loss/loss.py

```diff
@@ -757,6 +758,51 @@ def constant_to_optimal_zero(self, y_true, sample_weight=None):
             return term
 
 
+class HalfTweedieLossIdentity(BaseLoss):
+    """Half Tweedie deviance loss with identity link, for regression.
```

Comment:
This new loss class is needed for `TweedieRegressor(link="identity")`.
See Cython implementation in file `_loss.pyx.tp`.